build.ps1

```diff
@@ -705,6 +731,25 @@ function Build-WiXProject() {
   Invoke-Program $msbuild @MSBuildArgs
 }
 
+function Build-CompilerDependencies($Arch) {
+  Isolate-EnvVars {
+    $env:Path = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Runtimes\0.0.0\usr\bin;${env:Path}"
+    $env:SDKROOT = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Platforms\Windows.platform\Developer\SDKs\Windows.sdk"
+
+    Build-CMakeProject `
+      -Src $SourceCache\swift-syntax `
+      -Bin $BinaryCache\99 `
+      -InstallTo "$($Arch.ToolchainInstallRoot)\usr" `
+      -Arch $Arch `
+      -BuildTargets default `
+      -Defines @{
+        CMAKE_Swift_COMPILER = "$BinaryCache\toolchains\$PinnedToolchain\PFiles64\Swift\Toolchains\0.0.0+Asserts\usr\bin\swiftc.exe";
```

Comment:
Do we need to repeat the full path if it's in the Path above?

Reply:
Yes, we should always use absolute paths even if it is in the path.  This is super critical as we don't want accidental usage of the wrong compiler.  (`S:\b\1\bin` will always be given precedence).